events/importer/espoo.py

```diff
@@ -356,8 +362,13 @@ def _import_origin_objs(
     def origin_id(instance):
         return getattr(instance, instance_id_field)
 
+    queryset = model.objects.filter(data_source=data_source)
+
+    if model == Event:
+        queryset = queryset.filter(start_time__gte=EVENT_START)
```

Comment:
Just a theoretical question, but if the `end_date >= datetime.date.today()` then that event would not be updated?

Reply:
the `end_time` is irrelevant in the solution. Only `start_time` matter in this context so it will be updated if the `start_time` is greater or equal to the specified `EVENT_START`. If  `end_time >= datetime.date.today()` then the `start_time` would most likely also be and if start would be before `EVENT_START` it would not be updated regardless of the end time.